ai_worker/handler.py

```
@@ -0,0 +1,174 @@
+from typing import Dict, Any, Optional
+
+# Import AI Pipeline modules
+from src.parsers import (
+    ImageOCRParser,
+    ImageVisionParser,
+    PDFParser,
+    AudioParser,
+    VideoParser
+)
+from src.parsers.text import TextParser
+from src.storage.metadata_store import MetadataStore
```

Pylint (4.0.3)

[error] No name 'storage' in module 'src'

(E0611)